sage-ios/SAGE/SAGE/ProfileView.swift

```diff
@@ -30,7 +30,7 @@ class ProfileView: UIView {
     let userRoleLabel = UILabel()
     
     // profile page constants
-    let viewHeight = CGFloat(351.5)
+    let viewHeight = CGFloat(355)
```

Comment:
you need to move the loading indicator a bit up again its hitting the border lines